modules/statistics/client/views/statistics.client.view.html

```diff
@@ -85,6 +86,22 @@ <h5 class="text-muted text-uppercase">{{ stats.newsletter | number }} subscriber
 
         <hr>
 
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            <h3 class="panel-title">Member growth</h3>
+          </div>
+          <div class="panel-body">
+            <a href="https://grafana.trustroots.org/d/000000002/members?orgId=1">
+              <img src="https://grafana.trustroots.org/render/d-solo/000000002/members?orgId=1&from=1419289200000&to=1543142370362&panelId=1&width=800&height=300" width="100%>
```

Comment:
We should use responsive image class `img-responsive` here to make sure layout works nicely with all sizes &devices: https://getbootstrap.com/docs/3.3/css/#images-responsive